src/pages/start/7.nix-build-flake.mdx

```diff
@@ -187,7 +187,7 @@ Here's the package definition that enables us to build this Rust package:
 };
 ```
 
-For the full flake, see [`flake.nix`][flake-go] on GitHub or run `cat flake.nix`.
+For the full flake, see [`flake.nix`][flake-rust] on GitHub or run `cat flake.nix`.
```

Comment:
Thanks for this! I think you'll also need to add a link to the rust flake.nix at the bottom, with the rest of the links: https://github.com/DeterminateSystems/zero-to-nix/pull/192/files#diff-409f0076ad08e4d14c5a022825714a83e8fbfddbaa3bee76287b740a83c5c124R214

Reply:
Actually, it looks like there *is* a URL for the Rust flake but the alias is `flake-rs`. Update this and we're good to go!

Reply:
Ah, you're right -- I was expecting `flake-rust` and completely skimmed over `flake-rs` :laughing: 

```suggestion
For the full flake, see [`flake.nix`][flake-rs] on GitHub or run `cat flake.nix`.
```

Reply:
Thanks! @cole-h @lucperkins